core/jazz_ui/src/app/pages/environment-overview/env-overview-section.component.ts

```diff
@@ -249,15 +264,13 @@ popup(state){
     
   }
   onCancel(){
-    this.showCancel=false;
-    this.saveButton=false;
-    this.editButton=true;
-    this.isDiv=true;
-    this.showCncl=false;
-    this.saveBtn=false;
-    this.editBtn=true;
-    this.yaml='';
-    this.isEditClicked=false;
+    this.showCncl = false;
```

Comment:
showCncl different from showCancel? do we have both?

Reply:
yes

Reply:
Two edit button is available on that page.